README.md

```diff
@@ -100,6 +100,21 @@ _Note: SwiftGen needs Xcode 8.3 to build, so installing via Homebrew requires yo
 
 ---
 </details>
+<details>
+<summary>Via <strong>Mint</strong></summary>
+
+To install SwiftGen via [Mint](https://github.com/yonaskolb/Mint), simply use:
+
+Note: Will work for Releases > 5.3.0 only.
```

Comment:
I think this can be moved above the line above, so that the `simply use:` is followed by the example

Reply:
moved it to the end